src/Game/Managers/InfoBarManager.cs

```diff
@@ -81,11 +81,7 @@ public void Clear()
 
         public void Save()
         {
-            string path = Path.Combine
-            (
-                CUOEnviroment.ExecutablePath, "Data", "Profiles", ProfileManager.CurrentProfile.Username,
-                ProfileManager.CurrentProfile.ServerName, ProfileManager.CurrentProfile.CharacterName, "infobar.xml"
-            );
+            string path = Path.Combine(ProfileManager.ProfilePath, "infobar.xml");
```

Comment:
is `.ProfilePath` now the `ProfilesDirectory`? if possible can we keep the naming consistent?

Reply:
`ProfilesDirectory` is the directory containing all profiles (e.g.) `/path/to/classicUO/Data/Profiles/`.  `ProfilePath` is the path of the currently loaded profile (e.g.) `<ProfilesDirectory>/account/server/charactername/`.

Reply:
Its just the naming of the two things.
So there is 
`ProfilesPath` for `./Data/Profiles` and 
`ProfilePath` for `$(ProfilesPath)/account/server/character`

Then `ProfilesDirectory` should really be `ProfilesPath`.

Sorry for being picky here.

Reply:
renamed `directory` to `path`